packages/fuel-indexer-utils/src/utilities.rs

```diff
@@ -0,0 +1,37 @@
+use crate::prelude::sha256_digest;
+use crate::prelude::{Bytes32, SizedAsciiString};
```

Comment:
Could we import these from their actual module instead of the prelude? 

```rust
use fuel_indexer_plugin::types::{Bytes32, SizedAsciiString};
```

Just trying to keep the references as explicit as possible where possible